.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -40,10 +47,10 @@ stages:
     CACHE_MODE: none
     VERSION_CONTROL_OPTIONS: 'SONIC_VERSION_CONTROL_COMPONENTS='
   jobs:
-  - template: azure-pipelines-build.yml
+  - template: .azure-pipelines/azure-pipelines-build.yml@buildimage
```

Comment:
Can we only share the common tasks, like basic build tools installation, cleanup task, across branches? Some of platform builds better to place in each of the branch.